src/components/Editor/SessionList.vue

```diff
@@ -48,14 +51,16 @@
 						<span v-if="session.userId === null" class="guest-label">({{ t('text', 'guest') }})</span>
 					</li>
 				</ul>
-				<input id="toggle-color-annotations"
-					v-model="showAuthorAnnotations"
-					type="checkbox"
-					class="checkbox">
-				<label for="toggle-color-annotations">{{ t('text', 'Show author colors') }}</label>
-				<p class="hint">
-					{{ t('text', 'Author colors are only shown until everyone has closed the document.') }}
-				</p>
+				<form ref="form" tabindex="0" @submit.prevent>
```

Comment:
I think we should rather put the initial focus on https://github.com/nextcloud/text/pull/2771/files#diff-9e0a2a342f91d6277778b1bb6b17da4f37cdab38ef53c98a7c97a84deaf488a5R40 so that the content gets read and maybe give this a descriptive label then